### PR TITLE
Fix broken link

### PR DESCRIPTION
**Pull request by mayurkale22**
The property name change is as per, https://cloud.google.com/monitoring/api/resources#tag_gce_instance

### CODE REVIEW

File: utils/MonitoredResource.md [hunk 1]
```diff
@@ -6,21 +6,21 @@ detect these resources.
...
   * instance_id: The identifier for the GCE instance the container is running in.
   * pod_id: The identifier for the pod the container is running in.
   * container_name: The name of the container.
   * zone: The zone for the instance.
 * AWS_EC2_INSTANCE
-  * aws_account: The AWS account number for the instance.
+  * aws_account: The AWS account number for the VM.
   * instance_id: The VM instance identifier assigned by AWS.
-  * region: The AWS region for the cluster.
+  * region: The AWS region for the VM. The format of this field is "aws:{region}".
```
[the remark this answers is not in the document]
Comment (mayurkale22) [reply utils/MonitoredResource.md]: Removed.

File: utils/MonitoredResource.md [hunk 2]
```diff
@@ -6,21 +6,21 @@ detect these resources.
 ## Supported Monitored Resources
 
 * GCP_GCE_INSTANCE
-  * gcp_account: The GCP account number for the instance.
+  * project_id: The identifier of the GCP project associated with this resource.
```
Comment (bogdandrutu) [inline utils/MonitoredResource.md]: We are a generic library and should not follow the names in SD monitoring. Please revert this unless good arguments.

Comment (mayurkale22) [reply utils/MonitoredResource.md]: Done

**Comment by bogdandrutu** [comment]
Please rebase.

**Pull request merged**